mino/minogrpc/overlay.proto

```diff
@@ -18,4 +18,8 @@ service Overlay {
 
 message OverlayMsg {
     google.protobuf.Any message = 1;
+}
+
+message DummyMsg {
```

Comment:
Could you use a wrapper available? https://godoc.org/github.com/golang/protobuf/ptypes/wrappers
So we don't have a test message publicly available ?

Reply:
cool, I was precisely looking for something like that!